identity-aware-proxy/IAPClient/IAPClient.cs

```diff
@@ -62,16 +59,13 @@ public class IAPClient
     /// </summary>
     /// <param name="iapClientId">The client ID observed on 
     /// https://console.cloud.google.com/apis/credentials. </param>
-    /// <param name="credentialsFilePath">Path to the credentials .json file
-    /// downloaded from https://console.cloud.google.com/apis/credentials.
-    /// </param>
     /// <param name="cancellationToken">The token to propagate operation cancel notifications.</param>
     /// <returns>The HTTP response message.</returns>
-    public async Task<OidcToken> GetOidcTokenAsync(string iapClientId, string credentialsFilePath, CancellationToken cancellationToken)
+    public async Task<OidcToken> GetOidcTokenAsync(string iapClientId, CancellationToken cancellationToken)
     {
-        // Read credentials from the credentials .json file.
+        // Obtain the application default credentials.
         GoogleCredential credential = await GoogleCredential
-            .FromFileAsync(credentialsFilePath, cancellationToken).ConfigureAwait(false);
+            .GetApplicationDefaultAsync(cancellationToken).ConfigureAwait(false);

```

Comment:
Do we need the ConfigureAwait here? We don't have it elsewhere, and without that we could unwrap the line.

Reply:
I'll remove before merging. It's really mostly in case someone copy/pastes. But they should know...